### ISSUE TITLE
Mask output is incorrect

### ISSUE THREAD

**Issue by MeghaGhosh** [tymiao1220/TensorRT_Mask_rcnn#1]
Hi, thanks for this repository. It's very helpful. I have one issue, when I'm visualising the final colour splash or saving the mask(inference_engine.py), the output isn't correct. However, the confidence, boxes and classes are perfect. This is the output i'm getting. 

![image](https://user-images.githubusercontent.com/29752250/85207443-55653480-b346-11ea-81c5-bde8bf1153ca.png)

It would be great if you could help me solve the problem. Thank you!

**Comment by wlgjs8** [comment]
I get this same error :/
Did you solve it?

**Comment by MeghaGhosh** [comment]
Nope, I just went ahead and used the official TensorRT C++ inference code. Maybe you can use that to fix the python splash?

**Comment by wlgjs8** [comment]
Thanks for reply. ill try with official TensorRT C++ then :D
